.circleci/config.yml

```diff
@@ -205,25 +205,7 @@ jobs:
           command: |
             source ~/.bashrc
             pushd schema
-            sqitch deploy
-            sqitch verify
-            popd
-      - run:
-          name: regenerate schema via introspection
-          command: |
-            source ~/.bashrc
-            pushd app
-            yarn build:schema
-            popd
-      - run:
-          name: check for inconsistency between schema ddl & graphql
-          command: git diff --exit-code || (echo 'The contents of the `schema` folder are out of sync with `app/server/schema.graphql`. Did you forget to \"yarn build:relay\"?' && false)
```

Comment:
not sure what was the intended test by doing a rebase and regenerating the schema a second time

Reply:
it might be because relay's `schema.graphql` and `schema.json` are committed, so we're making sure they match our database schema from the same commit? But this should be caught by the e2e at least